annubes/tasks/task.py

```diff
@@ -59,8 +52,8 @@ class TaskSettings:
 
 
 @dataclass
-class Task:
-    """General class for defining a task.
+class Task(TaskSettingsMixin):
+    """General data class for defining a task.
```

Comment:
Maybe it would be good to define in the next line of the docstring what exactly we mean by a task.

Reply:
Good point. How do we define it?